README.md

```
@@ -442,18 +524,18 @@ func application(_ application: UIApplication, didReceiveRemoteNotification user
 
 >  ℹ️ Silent push notifications are not supported by the iOS simulator. To test silent push notifications, please use a real device.
 
-#### Handling background notifications (content-available)
+### Handling background notifications (content-available)
 
 Klaviyo can send a **standard** push (title, body, and other visible notification UI) whose APNs payload also includes **`content-available: 1`**. That is different from a [silent push](#silent-push-notifications): silent pushes never show an alert, while this is a normal user-visible notification that *additionally* asks iOS to wake your app in the background so you can refresh data or run other work from the same `userInfo`.
 
 You still handle the visible notification through [`UNUserNotificationCenterDelegate`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate) when the message is presented or opened. The background wake for `content-available` uses the same app delegate path as silent push: `application(_:didReceiveRemoteNotification:fetchCompletionHandler:)`. Implement that method (see the example under [Silent Push Notifications](#silent-push-notifications)) and finish by calling `completionHandler` with `.newData`, `.noData`, or `.failed` when your background work completes. Use the same Background Modes / remote-notification setup called out in that section (Apple’s [background updates](https://developer.apple.com/documentation/usernotifications/pushing-background-updates-to-your-app) guide).
 
 >  ℹ️ Background wakes are best-effort and may be throttled. As with [silent push notifications](#silent-push-notifications), test this functionality on a physical device because the Simulator does not support the full remote-notification background path.
```

markdownlint-cli2 (0.23.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)